src/map/utils/petutils.cpp

```diff
@@ -1005,6 +1005,385 @@ namespace petutils
             PPet->m_dmgType = DAMAGE_TYPE::IMPACT;
     }
 
+    void CalculateAvatarStats(CBattleEntity* PMaster, CPetEntity* PPet)
```

Comment:
Please confirm the substance of this function retains the values already set within the codebase.

Reply:
Everything that was previously changed in the pet stat calculation was moved to the new functions created by jmcmorris